packages/jest-console/src/BufferedConsole.ts

```diff
@@ -46,10 +48,20 @@ export default class BufferedConsole extends Console {
     type: LogType,
     message: LogMessage,
     level?: number | null,
-    sourceMaps?: SourceMapRegistry | null,
+    // TODO: remove in 26
+    _sourceMaps?: SourceMapRegistry | null
```

Comment:
TypeScript complains when callers pass the 5th parameter. Will try to modify all caller and thus removing the `getSourceMaps` from `BufferedConsole` to keep TypeScript from complaining.